CMakeLists.txt

```diff
@@ -137,8 +137,9 @@ endif()
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
 
 # Half
-find_package(HALF REQUIRED)
```

Comment:
@trixirt -- can you add this change in https://github.com/ROCm/rpp/blob/develop/cmake/FindHALF.cmake